internal/provider/resource_tfe_registry_module.go

```diff
@@ -113,6 +137,7 @@ func resourceTFERegistryModuleCreateWithVCS(v interface{}, meta interface{}, d *
 	// Create module with VCS repo configuration block.
 	options := tfe.RegistryModuleCreateWithVCSConnectionOptions{}
 	vcsRepo := v.([]interface{})[0].(map[string]interface{})
+	testConfig := v.([]interface{})[0].(map[string]interface{})
```

Comment:
`v.([]interface{})[0].(map[string]interface{})` typecasting is already done by vcsRepo just above in lines #139.

Also, v represents `vcs_repo` field, since, test_config is not a field in `vcs_repo`, it has to be extracted from the resourceData variable.

Reply:
I changed this.